src/results/financial.jl

```diff
@@ -117,10 +120,10 @@ function add_financial_results(m::JuMP.AbstractModel, p::REoptInputs, d::Dict; _
     
     r["lifecycle_capital_costs"] = value(m[Symbol("TotalTechCapCosts"*_n)] + m[Symbol("TotalStorageCapCosts"*_n)] + m[Symbol("GHPCapCosts"*_n)] +
         m[Symbol("OffgridOtherCapexAfterDepr"*_n)] - m[Symbol("AvoidedCapexByGHP"*_n)] - m[Symbol("ResidualGHXCapCost"*_n)] - m[Symbol("AvoidedCapexByASHP"*_n)])
-    
+    #TODO: Should lifecycle_capital_costs include m[:mgTotalTechUpgradeCost] + m[:dvMGStorageUpgradeCost]? 
```

Comment:
I'd say yes to this - the MG upgrade is also an upfront capital cost.

Reply:
Thanks for weighing in! I've added this here and in the initial capex no incentives function https://github.com/NREL/REopt.jl/pull/491/commits/3431afd9816909149eabee6034f16d486793793d